kclvm/compiler/src/codegen/llvm/context.rs

```diff
@@ -1320,6 +1320,14 @@ impl<'ctx> LLVMCodeGenContext<'ctx> {
                 self.pkgpath_stack.borrow_mut().push(pkgpath.clone());
             }
         }
+        // Set the kcl workdir to the runtime context
+        self.build_void_call(
+            &ApiFunc::kclvm_context_set_kcl_workdir.name(),
+            &[
+                self.current_runtime_ctx_ptr(),
+                self.native_global_string_value(&self.program.root),
```

Comment:
Why work_dir is the program root?

Reply:
fixed by https://github.com/kcl-lang/kcl/pull/1056